src/lang/english.txt

```diff
@@ -2835,7 +2835,7 @@ STR_BUILD_DEPOT_TRAM_ORIENTATION_SELECT_TOOLTIP                 :{BLACK}Select t
 STR_STATION_BUILD_BUS_ORIENTATION                               :{WHITE}Bus Station Orientation
 STR_STATION_BUILD_BUS_ORIENTATION_TOOLTIP                       :{BLACK}Select bus station orientation
 STR_STATION_BUILD_TRUCK_ORIENTATION                             :{WHITE}Lorry Station Orientation
-STR_STATION_BUILD_TRUCK_ORIENTATION_TOOLTIP                     :{BLACK}Select lorry loading bay orientation
+STR_STATION_BUILD_TRUCK_ORIENTATION_TOOLTIP                     :{BLACK}Select lorry station orientation
```

Comment:
Your description says one thing, and your patch does another.

But do we actually want to invalidate translations? Is that actually worth it? I mean .. translations that won't get to updating it, will have an english text all of a sudden, and possibly in most translations it is already translated consistently. But with this rename, it gets entered as a new string, and not an updated, losing that translation.

I don't have an answer, just a wonder: do we actually want to do this? But most of all, if you claim you rename to invalidate translations, do it consistently, and not 3 out of 4 :) That makes it looks like you weren't actually after invalidating, but you just wanted the STR name to be "correct" :)

Reply:
Just to be clear: if you do not rename the STR, translators will see the english text was updated, and they need to reconsider their translation. BUT. The old translation remains there.

What you do with the other 3 strings will remove the translation completely, forcing it back to english, until they retranslated it again.

Reply:
Yes, this one doesn't get deleted but will be flagged as Outdated.

The ideal outcome is Outdated for all three, but the only way to do that is to leave the string names alone, which feels sloppy to leave for someone to clean up later...

Reply:
So for your own OCD, you will make the life of .. 30+ people harder? Seems like the wrong balance :)

Many STR strings get out-of-sync with their translation over time. For 13.0 we did this:
https://github.com/OpenTTD/OpenTTD/pull/10121

That might be the better approach. Not in this PR. But just a PR to rename everything we consider "wrong". Worked last time at least. And punishes our translators a lot less.

Reply:
Ah, so if we change string values and names in two steps with an eints run in the middle, we can avoid deleting them? Sounds ideal!

Also, you taught me to be OCD! 😛

Reply:
No, eints has little to do with that. With #10121, all translations were renamed too. The difference with the approach is:

Translators will have a few weeks / months to see the english text changed, and can make adjustments if needed. We then rename the string for them, and eints is like: owh new string, how lovely, and adds it to the list of "maybe look at this", without history. So translators wouldn't know why it is there. Which is fine, as we can tell them: yo, we renamed these N strings.

More the point is: in this PR, you just want to fix the english text. Not the STR. That is our internal bookkeeping. As I said on Discord:

Making user-facing things consistent: good idea
Making things for us consistent but giving translators a lot of work: bad idea (imo)

We can live with "LORRY_STATION"; we know what it means. And if we have a list long enough to do a mass renaming, sure, go for it :)

https://github.com/OpenTTD/OpenTTD/discussions/9101 was an excellent idea to track those strings btw. We might want to start one up again, for 15.0 or something.

Reply:
Ah, thanks.

If/when this gets merged, I will start a Discussion for 15.0 and add these strings.